README.md

```diff
@@ -116,6 +116,11 @@ Output (Stripe):
         "donating_company": null,
         "currency": "USD",
         "converted_currency": "USD",
```

Comment:
@fabes did we deploy a `utm_utm_campaign` attribute? I was strictly specifc about the name of the attributes: https://github.com/donorbox/donorbox/pull/3633#issuecomment-501918025.

APIs demans careful design, because once deployed, they can break external integrations.

Please create a bugfix as your highest priority today.

Also fix this code style (DRY principle!)

## Bad

```
  delegate :utm_source, to: :utm_campaign, allow_nil: true
  delegate :utm_medium, to: :utm_campaign, allow_nil: true
  delegate :utm_campaign, to: :utm_campaign, allow_nil: true, prefix: :utm
  delegate :utm_term, to: :utm_campaign, allow_nil: true
  delegate :utm_content, to: :utm_campaign, allow_nil: true
```

## Good
```
  delegate :utm_source, :utm_medium, :utm_term, :utm_content, 
    :utm_campaign, to: :utm_campaign, allow_nil: true
```

Reply:
Thank you for the suggestion @fabiolnm! I would like to confirm if renaming this attribute will affect the Salesforce mappings before creating a fix.

Reply:
@harish86 @kyleevert do you know if changing the names will affect the mapping?

Reply:
@rebelvc @fabes 
Fabio's suggestion for naming attributes are same as before except `utm_campaign` attribute.
It has prefix `utm` before, so the attribute name is `utm_utm_campaign`.
But the new code will be just `utm_campaign` name for this attribute.

So I suggest to fix the code style like this:
```
delegate :utm_source, :utm_medium, :utm_term, :utm_content, 
    to: :utm_campaign, allow_nil: true
delegate :utm_campaign, to: :utm_campaign, allow_nil: true, prefix: :utm
```
This code is same as before.

Reply:
@fabiolnm @kyleevert so without prefix it is going to raise an exception because we have column name same as the model association name. I think we should change the association name and update existing SF mapping fields in our database
```
belongs_to :utm_params, class_name: "UtmCampaign", foreign_key: :utm_campaign_id
```